#request

# true:on
# false: off
bool power

---
#response

bool success